### DISCUSSION TITLE
MT Files Exportation

### DISCUSSION THREAD

**Discussion by Yannos60** [mytourbook/mytourbook#1382] [Q&A] [answered]
Bonjour, since last update I only can export .mt Files one by one, not several at the same time. If you have an explaination I'll be happy to get it. Merci

**Comment by FJBDev** [comment]
@Yannos60 Can you give more details ? I don't understand your request/issue.

**Comment by Yannos60** [comment]
@ FJB Bonjour, Better than Words,see screen captures below.
![1](https://github.com/user-attachments/assets/8a8eaaa6-91fd-4b02-ab78-5819ba00ad79)
![2](https://github.com/user-attachments/assets/0f9abe53-0237-4457-aea2-56e44ecd95d9)
![3](https://github.com/user-attachments/assets/8a4f34d7-59e5-4152-acc2-29e32b297e22)
In fact, when I want to export only one track in MTB , mt format : It works.
When I select several, I get the error message : " This is not a UI Thread (Or device already disposed)  Java.langNullPointerException.
I do this export to save my tracks as, some months ago, I had to reinstall windows and I lost some files. 

Thanks for your help.

Yannos

**Comment by wolfgang-ch** [comment, marked as answer]
This issue is fixed in the next version, it happened when multiple tours are exported

https://github.com/mytourbook/mytourbook/commit/9a49beda2e95c388bc2b59367bcb9e1a481b1fd1 https://github.com/mytourbook/mytourbook/issues/1383

**Reply by Yannos60** [reply]
Thanks !
